src/components/src/ProductListItem/productlistitem.main.tsx

```diff
@@ -147,8 +147,14 @@ class ProductListItemMain extends Component<ProductListItemMainProps, ProductLis
       return (
         <div className="category-item-inner">
           <div className={`category-item-thumbnail-container ${imageStatus === 'loaded' ? 'loaded' : ''}`}>
-            <Link to={`${itemDetailLink}/${encodeURIComponent(productData._code[0].code)}`} title={productData._definition[0]['display-name']}>
```

Comment:
axe reports an error due to the missing title. I'll fix this in my PR with the version update before release cut-off:
https://github.com/elasticpath/react-pwa-reference-storefront/pull/712